integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/AllergyIntoleranceFhirResourceProviderIntegrationTest.java

```diff
@@ -514,4 +514,39 @@ public void shouldSearchForExistingAllergyAsXML() throws Exception {
 		
 	}
 	
+	@Test
+	public void shouldReturnCountForAllergyIntoleranceAsJson() throws Exception {
+		MockHttpServletResponse response = get("/AllergyIntolerance?_summary=count").accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, isOk());
+		assertThat(response.getContentType(), is(FhirMediaTypes.JSON.toString()));
+		assertThat(response.getContentAsString(), notNullValue());
+		
+		System.out.println(response.getContentAsString());
```

Comment:
We don't really need these in tests. Please remove them from all the added integration tests.

Reply:
@varung-31 Do I need to remove the 3 tests on the response also or just the System.out.println part?

Reply:
The `System.out.println()` part. The earlier tests are just to provide a better error message if the testing framework failed for whatever reason.

Reply:
@ibacher @varung-31 I'll remove it. I just left it in this class by mistake.